### ISSUE TITLE
Elevated alveolar-arterial (A-a) oxygen gradient

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11015]
**Preferred term label:**
Elevated alveolar-arterial (A-a) oxygen gradient

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.mdcalc.com/calc/243/a-a-o2-gradient
https://www.ncbi.nlm.nih.gov/books/NBK545153/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pulmonary fibrosis
ARDS
Pulmonary embolism
Pneumonia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001469
- label: Elevated alveolar-arterial (A-a) oxygen gradient
- parents: [Abnormal blood oxygen level (HP:0500165)](https://hpo.jax.org/app/browse/term/HP:0500165)
- pmid: PMID:31424737
- def: An increase in the difference between the oxygen concentration in the alveoli and arterial system above the upper limit of normal.
- comment: The A-a gradient, or the alveolar-arterial gradient, measures the difference between the oxygen concentration in the alveoli and arterial system. The A-a gradient has important clinical utility as it can help narrow the differential diagnosis for hypoxemia. The A-a gradient calculation is as follows:  A-a Gradient = PAO2 – PaO2. In this equation, PAO2 represents alveolar oxygen pressure and PaO2 representing arterial oxygen pressure. The arterial oxygen pressure (PaO2) can be directly assessed with an arterial blood gas test (ABG) or estimated with a venous blood gas test (VBG). The alveolar oxygen pressure (PAO2) is not easily measured directly; instead, it is estimated using the alveolar gas equation. The A-a gradient may be increased in clinical conditions such as pneumonia, with poor diffusion of oxygen across the alveolar-capillary unit and thus lower oxygen levels in the arterial blood (because of the physical barrier within the alveoli, limiting the diffusion of oxygen into the capillaries).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11015
- synonyms:
